bitnami/mysql/README.md

```diff
@@ -105,6 +105,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `initdbScriptsConfigMap`           | ConfigMap with the initdb scripts (Note: Overrides `initdbScripts`)                                                                                                                 | `""`                    |
 | `startdbScripts`                   | Dictionary of startdb scripts                                                                                                                                                       | `{}`                    |
 | `startdbScriptsConfigMap`          | ConfigMap with the startdb scripts (Note: Overrides `startdbScripts`)                                                                                                               | `""`                    |
+| `createSecret`                     | Create secrets for passwords according configuration. Set to false if you use secret injection like vault-injector.                                                                 | `true`                  |
```

Comment:
This is not a parameter in the values.yaml file so it can't be added here. It's a function [defined in the helpers file](https://github.com/bitnami/charts/blob/main/bitnami/mysql/templates/_helpers.tpl#L139)